### PR TITLE
New focal loss

### PR DESCRIPTION
**Pull request by rahulmohan**

### CODE REVIEW

File: variantworks/utils/losses.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Custom loss function implementations."""
+
+
+import torch
+from torch.nn.functional import one_hot
```
Comment (tijyojwad) [inline variantworks/utils/losses.py:21]: `one_hot` doesn't seem to be used anywhere

File: variantworks/utils/losses.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import torch.nn.functional as F
+from nemo.backends.pytorch.nm import LossNM
+from typing import Optional
```
Comment (tijyojwad) [inline variantworks/utils/losses.py:24]: we haven't standardized the use of `Optional` in our repo yet, so let's remove that usage as well

**Review by tijyojwad — CHANGES_REQUESTED** [review]
Overall looks good. CI is failing, so once that's fixed I can approve and merge

**Review by tijyojwad — APPROVED** [review]
